src/test/java/com/meilisearch/integration/TasksTest.java

```diff
@@ -202,6 +205,38 @@ public void testClientGetTasksAllQueryParameters() throws Exception {
         assertThat(result.getNext(), is(notNullValue()));
         assertThat(result.getResults().length, is(notNullValue()));
     }
+    /** Test to check if task list is reversed when enabled */
+    @Test
+    public void testGetTasksInReverse() {
+        String indexUid = "tasksOnReverseOrder";
+        Date currentTime = Date.from(Instant.now());
+        TestData<Movie> testData = this.getTestData(MOVIES_INDEX, Movie.class);
+
+        TaskInfo taskA = client.index(indexUid).addDocuments(testData.getRaw());
+        TaskInfo taskB = client.index(indexUid).addDocuments(testData.getRaw());
+
+        client.waitForTask(taskA.getTaskUid());
```

Comment:
If you wait for the last task the first one will be finished in any case so you can remove this line.